### DISCUSSION TITLE
Where is battle.net saved to?

### DISCUSSION THREAD

**Discussion by Timmybread22** [moraroy/NonSteamLaunchers-On-Steam-Deck#65] [Q&A] [answered]
I would like to add Diablo 4 as a non-steam game. Where can I find the file in Linux to add it?

**Comment by daviator** [comment, marked as answer]
You can find the path to the prefix folder under the properties of the BattleNet launcher. I would recommend to always start through the BattleNet launcher though, I am not sure how stable it is to launch Diablo 4, which then needs to launch BattleNet in the background in order to work. 
The code from NonSteamLaunchers.sh:
`battlenet_path1="$HOME/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program Files (x86)/Battle.net/Battle.net Launcher.exe"`
and
`battlenet_path2="$HOME/.local/share/Steam/steamapps/compatdata/Battle.netLauncher/pfx/drive_c/Program Files (x86)/Battle.net/Battle.net Launcher.exe"`

For me it was:
`"$HOME/.local/share/Steam/steamapps/compatdata/NonSteamLaunchers/pfx/drive_c/Program Files (x86)/Diablo IV`
